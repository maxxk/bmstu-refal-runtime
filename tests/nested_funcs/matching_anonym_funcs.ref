$ENTRY Go
{
	= <Prout <Func {=;} {=;}>>	
}

Func
{
	s.1 s.2 : s.2 s.1 = Success;
}
