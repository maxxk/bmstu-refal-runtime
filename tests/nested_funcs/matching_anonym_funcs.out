[VM]: Func call failed!
Func call term:
<Func AnonymFunc_1 AnonymFunc_2 >
Field of view:
<Prout <Func AnonymFunc_1 AnonymFunc_2 >>
